Makefile

```diff
@@ -8,6 +8,8 @@ include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
 	targets/openshift/deps.mk \
 )
 
+export GO_BUILD_FLAGS="-gcflags=all=-N -l"
```

Comment:
It is mandatory to use correct `gcflags` to let Delve successfully debug the binary.